README.md

```diff
@@ -10,10 +10,24 @@ Ruby LSP Rails is a [Ruby LSP](https://github.com/Shopify/ruby-lsp) extension fo
 To install, add the following line to your application's Gemfile:
 
 ```ruby
+# Gemfile
 group :development do
   gem "ruby-lsp-rails"
 end
 ```
+Some features rely on server introspection, and use a Rack server which is automatically mounted by using a Railtie.
+
+For applications with specialized routing requirements, such as custom sharding, this may not be compatible. It can
+be disabled with:
+
+```ruby
+# config/environments.development.rb
+Rails.application.configure do
+  # ...
+  config.x.ruby_lsp_rails.server = false
```

Comment:
Using `config.x` as per the docs recommendation:

https://guides.rubyonrails.org/configuring.html

> The key difference between these two is that you should be using config.x if you are defining nested configuration (ex: config.x.nested.hi), and just config for single level configuration (ex: config.hello).

Reply:
`x` is only for application config. Libraries should expose their own configs.